docs-devsite/ai.imagenmodel.md

```diff
@@ -144,7 +144,7 @@ If the request to generate images fails. This happens if the prompt is blocked.
 const imagen = new ImagenModel(
   ai,
   {
-    model: 'imagen-3.0-generate-002'
+    model: 'imagen-model-name'
```

Comment:
Is it clear and understood that this should be replaced with an actual model name?